doc/develop/test/ztest.rst

```diff
@@ -145,28 +149,28 @@ Test rules
 Test rules are a way to run the same logic for every test and every suite. There are a lot of cases
 where you might want to reset some state for every test in the binary (regardless of which suite is
 currently running). As an example, this could be to reset mocks, reset emulators, flush the UART,
-etc.
+etc.:
```

Comment:
Is this added `:` necessary?

Reply:
> Is this added : necessary?

Generally speaking I would expect to see a colon between the end of any "for example" type clause and the example itself. This is consistent with the rest of the file.

Reply:
Good point.